cpp/velox/utils/ConfigExtractor.cc

```diff
@@ -189,6 +189,20 @@ std::shared_ptr<facebook::velox::config::ConfigBase> getHiveConfig(
   }
 #endif
 
+#ifdef ENABLE_ABFS
+  std::string_view kSparkHadoopPrefix = "spark.hadoop.";
+  std::string_view kSparkHadoopAbfsPrefix = "spark.hadoop.fs.azure.";
+  std::string_view kAbfsPrefix = "fs.azure.";
+  for (const auto& [key, value] : conf->rawConfigs()) {
+    if (key.find(kAbfsPrefix) == 0) {
```

Comment:
Let's always use prefix spark.hadoop.fs.azure. in Gluten to align with Spark's configuration.

Reply:
done!